forge/src/main/java/org/openjdk/skara/forge/HostedRepository.java

```diff
@@ -208,4 +208,10 @@ default URI reviewUrl(Hash hash) {
     default boolean isSame(HostedRepository other) {
         return name().equals(other.name()) && forge().name().equals(other.forge().name());
     }
+
+    /**
+     * Delete deploy keys which are older than 'age' in this repository
+     * The return value is the count of deleted keys
+     */
+    int deleteDeployKeys(Duration duration);
```

Comment:
Should probably call the parameter `age`.

Reply:
Ok, will fix it soon